test/test.js

```diff
@@ -1765,3 +1765,30 @@ test( "Validation triggered on radio and checkbox via click", function() {
 	// test if there is no error anymore
 	ok( form.find( "input.error" ).length === 0, "Form valid" );
 });
+
+test( "destruction of Validator", function() {
+	expect( 2 );
+
+	var form = $( "#validatorDestroy" ),
+		validate = form.validate();
+
+	form.on( "submit.validate", function( event )
+	{
+		event.preventDefault();
+
+		ok( !!$( this ).data( "validator" ), "Validator has events and object" );
+	});
+
+	form.submit();
+
+	validate.destroy();
+
+	form.on( "submit.validate", function( event )
```

Comment:
The exact error was something along the lines of 'unable to access property of undefined' or something along those lines - the :submit/:input handler was still set, and when calling `$(form).trigger("submit")` it would cause some of the old handlers to try to access the validator object, which, at this time was no longer in existence.